spec/conversions.md

```diff
@@ -59,246 +59,255 @@ The following conversions are classified as implicit conversions:
 
 Implicit conversions can occur in a variety of situations, including function member invocations (§12.6.6), cast expressions (§12.8.7), and assignments (§12.18).
 
-The pre-defined implicit conversions always succeed and never cause exceptions to be thrown. \[*Note*: Properly designed user-defined implicit conversions should exhibit these characteristics as well. *end note*\]
+The pre-defined implicit conversions always succeed and never cause exceptions to be thrown. 
+>[!NOTE]
+>Properly designed user-defined implicit conversions should exhibit these characteristics as well. 
 
-For the purposes of conversion, the types object and dynamic are considered equivalent.
+For the purposes of conversion, the types `object` and `dynamic` are considered equivalent.
 
-However, dynamic conversions (§11.2.9 and §11.3.7) apply only to expressions of type dynamic (§9.2.4).
+However, dynamic conversions (§11.2.9 and §11.3.7) apply only to expressions of type `dynamic` (§9.2.4).
 
 ### Identity conversion
 
 An identity conversion converts from any type to the same type. One reason this conversion exists is so that a type T or an expression of type T can be said to be convertible to T itself.
 
-Because object and dynamic are considered equivalent there is an identity conversion between object and dynamic, and between constructed types that are the same when replacing all occurrences of dynamic with object.
+Because `object` and `dynamic` are considered equivalent there is an identity conversion between `object` and `dynamic`, and between constructed types that are the same when replacing all occurrences of `dynamic` with `object`.
 
 In most cases, an identity conversion has no effect at runtime. However, since floating point operations may be performed at higher precision than prescribed by their type (§9.3.7), assignment of their results may result in a loss of precision, and explicit casts are guaranteed to reduce precision to what is prescribed by the type.
 
 ### Implicit numeric conversions
 
 The implicit numeric conversions are:
 
--   From sbyte to short, int, long, float, double, or decimal.
+-   From ``sbyte`` to `short`, `int`, `long`, `float`, `double`, or `decimal`.
```

Comment:
doubled back-ticks should be single:

```suggestion
-   From `sbyte` to `short`, `int`, `long`, `float`, `double`, or `decimal`.
```

Reply:
@BillWagner I have made this fix since @chamukallem-msft is OOF for the rest of the week. You can remove conversions.md from the merge in this PR and use the version here: https://github.com/dotnet/csharplang/pull/3199 instead. Thanks!